src/libraries/System.Globalization/tests/IcuTests.cs

```diff
@@ -10,8 +10,11 @@ namespace System.Globalization.Tests
     public class IcuTests
     {
         private static bool IsIcuCompatiblePlatform => PlatformDetection.IsNotWindows ||
-                                                       PlatformDetection.IsWindows10Version1903OrGreater;
+                                                       (PlatformDetection.IsWindows10Version1903OrGreater &&
+                                                        // Server core doesn't have icu.dll on SysWOW64
+                                                        !(PlatformDetection.IsWindowsServerCore && PlatformDetection.IsX86Process));
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/64763", typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindows), nameof(PlatformDetection.IsX86Process))]
```

Comment:
Do we need to include this active issue attribute now that you are including it to the condition that runs the test? 

Also, your PR is taking care of the issue so I wonder if we still need it.

Reply:
Sorry, that was unintentionally included in the commit (from an earlier attempt to fix the original issue). I've removed it already.